Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.cpp

```diff
@@ -317,6 +328,20 @@ namespace AzToolsFramework
         return false;
     }
 
+    AZ::s32 QtEventToAzInputMapper::GetPriority() const
+    {
+        return AzFramework::InputChannelEventListener::GetPriorityLast();
+    }
+
+    void QtEventToAzInputMapper::OnInputChannelEvent(const AzFramework::InputChannel& inputChannel,
+                                                     bool& o_hasBeenConsumed)
```

Comment:
update: `o_` I do not believe is something in the coding standards (nor is `out`). This would be better as just `hasBeenConsumed`, the type information alone is enough to let us know this can be modified by the caller. Also here we use `o_` but this is an in/out parameter as we read it here and never write to it, so the 'hungarian-style' notation is not accurate. It would be good at the API to call out this is an inout parameter and leave the name as just `hasBeenConsumed`.

nit: Please can you run clang-format on this file, it does not use this alignment style and so will cause unnecessary changes on the next edits to this file (we have already run it on these files. You can do this from Visual Studio by running `Ctrl-K, D`. Thanks!

Reply:
For clarify, the naming style came from the input API directly but will update here accordingly.

Reply:
Okay perfect thanks, apologies I wasn't aware of that (potentially something we can update in the API in future)